README.md

```diff
@@ -117,14 +117,84 @@ This is to ensure setup was successful
 Open a browser and navigate to `localhost:3000` to view the documentation site on your local system
 
 #### Problems?
-Running a build on first install is to ensure that all tools have been installed and are reading the code correctly. Please make sure to debug any errors at this point until you can get a full preview of the docs site at localhost:3000 in your browser. 
+Running a build on first install is to ensure that all tools have been installed and are reading the code correctly. Please make sure to debug any errors at this point until you can get a full preview of the docs site at localhost:3000 in your browser.
 
 :TODO: image localhost docs landing page
 
 ### Stop the local preview server
 This necessary once you are done previewing, not great to leave it running.
 `ctrl-c`
 
+## Deploy Live Site in a Container
```

Comment:
I didn't even know this file existed. We have been using the Wiki for this information. I need to do a cross reference or make sure this new info is in the wiki somewhere.